mui/dockwidgets/run_dialog.py

```diff
@@ -0,0 +1,241 @@
+import shlex
+from typing import List, Callable
+
+from PySide6.QtCore import Qt
+from PySide6.QtWidgets import (
+    QDialog,
+    QWidget,
+    QVBoxLayout,
+    QLabel,
+    QHBoxLayout,
+    QPushButton,
+    QLineEdit,
+    QCheckBox,
+    QFormLayout,
+    QFileDialog,
+    QScrollArea,
+    QLayout,
+)
+from binaryninja import (
+    BinaryView,
+    show_message_box,
+    MessageBoxButtonSet,
+    MessageBoxIcon,
+    Settings,
+    SettingsScope,
+)
+from binaryninjaui import UIContext
+
+
+class ListWidget(QWidget):
+    def __init__(
+        self,
+        parent: QWidget = None,
+        initial_row_count: int = 0,
+        validate_fun: Callable = lambda: None,
```

Comment:
Would this be a more accurate type annotation?
```suggestion
        validate_fun: Callable[[], None] = lambda: None,
```